styles/styles.css

```
@@ -0,0 +1,610 @@
+    color:#FFFFFF;
+    font-size: 20px;
+}
+
+.main-index {
+    background-color: #FFFFFF;
+}
+
+.welcome {
+    background-image: url(../images/index-background.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: top;
+}
+
+.welcome-text {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+
+
+.advantages-service-list {
+    display: flex;
+}
+
+
+.advantages-item-first,
+.advantages-item-fifth {
+    background-color: #82B3D3;
+    color:#FFFFFF;
+}
+
+.advantages-item-second,
+.advantages-item-fourth {
+    background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-third {
+    background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-service-item {
+    width: 400px;
+}
+
+.advantages-service-item-first,
+.advantages-service-item-third {
+    background-color: rgba(131, 179, 211, 0.12);
+    color:#FFFFFF;
+}
+
+.advantages-service-item-first {
+    background-image: url(../images/house.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.advantages-service-item-second {
+    background-image: url(../images/food.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.advantages-service-item-third {
+    background-image: url(../images/present.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.newsletter-background {
+    background-image: url(../images/catalog-background-1-redactor.jpg);
+    color: #FFFFFF;
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: 0%;
+    background-color: #82B3D3;
+}
+
+.newsletter,
+.newsletter-catalog {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+
+
+
+.contacts-phone {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 40px;
+    color: #000000;
+    text-decoration: none;
+    margin-top: 45px;
+    margin-left: 448,5px;
+    margin-right: 420,5px;
+    margin-bottom: 35px;
+}
+
+.catalog-background-header {
+    background-image: url(../images/catalog-background-1-redactor.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: 0%;
+    color: #FFFFFF;
+}
+
+.catalog-title {
+    font-family: 'PT Sans';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 78px;
+}
+
+.catalog-link {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+}
+
+.catalog-filter-infrastructure,
+.catalog-type-of-hotels {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    text-transform: capitalize;
+}
+
+.infrastructure-list,
+.type-hotel {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 23px;
+}
```

```suggestion
.infrastructure-list,
.type-hotel {
    font-weight: 400;
    font-size: 18px;
    line-height: 23px;
}
```

И тут тоже можно задать одинаковые классы для одинаковых свойств. И классы-модификаторы для разных.